### PR TITLE
better README.md

### PR DESCRIPTION
**Pull request by pannous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
...
 * [python sox](https://pypi.python.org/pypi/sox)
 
+
+## Install
+
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
+```
+git clone https://github.com/mozilla/DeepSpeech
+cd DeepSpeech
+git lfs fetch
```
Comment (kdavis-mozilla) [inline README.md]: This does not work for me on OS X, see below. When I clone, the large files are already downloaded.

```bash
kdaviss-MBP:DeepSpeech kdavis$ git lfs fetch

Usage: 
  git-lfs [flags]
  git-lfs [command]

Available Commands: 
  clean                     Implements the Git clean filter
  env                       Show the current environment
  init                      Initialize the default Git LFS configuration
  logs                      View error logs
  ls-files                  Show information about Git LFS files
  pointer                   Build and compare pointers between different Git LFS implementations
  push                      Push files to the Git LFS server
  smudge                    Implements the Git smudge filter
  status                    Show information about Git LFS objects that would be pushed
  track                     Manipulate .gitattributes
  untrack                   Remove an entry from .gitattributes
  update                    Update local Git LFS configuration
  version                   Show the version number
  pre-push                  Implements the Git pre-push hook
  help [command]            Help about any command

 Available Flags:
      --help=false: help for git-lfs

Use "git-lfs help [command]" for more information about that command.
Error: unknown command "fetch"
Run 'help' for usage.

git-lfs: invalid command [`fetch`]
Run 'git-lfs help' for usage
```

File: README.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
...
+git lfs checkout
```
Comment (kdavis-mozilla) [inline README.md]: This does not work for me on OS X, see below. When I clone, the large files are already downloaded.

```bash
kdaviss-MBP:DeepSpeech kdavis$ git lfs checkout

Usage: 
  git-lfs [flags]
  git-lfs [command]

Available Commands: 
  clean                     Implements the Git clean filter
  env                       Show the current environment
  init                      Initialize the default Git LFS configuration
  logs                      View error logs
  ls-files                  Show information about Git LFS files
  pointer                   Build and compare pointers between different Git LFS implementations
  push                      Push files to the Git LFS server
  smudge                    Implements the Git smudge filter
  status                    Show information about Git LFS objects that would be pushed
  track                     Manipulate .gitattributes
  untrack                   Remove an entry from .gitattributes
  update                    Update local Git LFS configuration
  version                   Show the version number
  pre-push                  Implements the Git pre-push hook
  help [command]            Help about any command

 Available Flags:
      --help=false: help for git-lfs

Use "git-lfs help [command]" for more information about that command.
Error: unknown command "checkout"
Run 'help' for usage.

git-lfs: invalid command [`checkout`]
Run 'git-lfs help' for usage
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
...
+sudo pip install -r doc/requirements.txt
```
Comment (kdavis-mozilla) [inline README.md]: We shouldn't require root access, people may need to work on a box where they can't sudo.

Also, and this is my fault, the kenlm install `https://github.com/kpu/kenlm/archive/master.zip` is missing from `doc/requirements.txt`. Could you add it there?

Comment (pannous) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (pannous) [inline README.md]: confirmed that the lfs commands are unnecessary if git-lfs  is installed before hand. Removed.

File: README.md [hunk 4]
```diff
@@ -11,6 +11,16 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
...
 * [python sox](https://pypi.python.org/pypi/sox)
 
+
+## Install
+
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
+```
+git clone https://github.com/mozilla/DeepSpeech
+cd DeepSpeech
+sudo pip install -r doc/requirements.txt
```
Comment (kdavis-mozilla) [inline README.md]: Could you remove the `sudo`call. We don't want to require root access.

**Comment by kdavis-mozilla** [comment]
Thanks!

**Pull request merged**